lib/endpoints/class-acf-to-rest-api-controller.php

```diff
@@ -229,6 +236,24 @@ protected function get_fields( $request, $response = null, $object = null ) {
 			return apply_filters( 'acf/rest_api/' . $this->type . '/get_fields', $data, $request, $response, $object );
 		}
 
+		protected function add_relation_fields( &$acf_data, $field ) {
+			foreach ( $acf_data as $k => $rel_array ) {
```

Comment:
Nope. You must have check the **$rel_array** variable.
When I said: "you need to pass the values by reference" in this case also I refer the **$rel_array**. https://github.com/airesvsg/acf-to-rest-api/pull/95#issuecomment-269369868
I prefer that the method `add_relation_fields` returns the data.
Thanks